jarbas/core/management/commands/reimbursements.py

```diff
@@ -80,16 +77,16 @@ def serialize(self, reimbursement):
 
         return reimbursement
 
-    def bulk_create_by(self, reimbursements, size):
-        batch = list()
-        for reimbursement in reimbursements:
-            batch.append(reimbursement)
-            if len(batch) == size:
-                self.bulk_create(batch)
-                batch = list()
-        self.bulk_create(batch)
-
-    def bulk_create(self, batch):
-        Reimbursement.objects.bulk_create(batch)
-        self.count += len(batch)
-        self.print_count(Reimbursement, count=self.count)
+    def create_or_update(self, reimbursements_as_dicts):
+        for count, reimbursement in enumerate(reimbursements_as_dicts):
+            document_id = reimbursement.get('document_id')
+            if document_id:
+                Reimbursement.objects.update_or_create(
+                    document_id=document_id,
+                    defaults=reimbursement
+                )
+            self.print_count(Reimbursement, count=count + 1)
```

Comment:
Yes. It's only a metter of usability. Since print_count() default is created objects, I suppose the users are used to take that log as "created objects". Since here it's not the same info, but with same output, I got worried about some misunderstanding.